lte/gateway/fabfile.py

```diff
@@ -384,7 +384,7 @@ def bazel_integ_test_post_build(
         ansible_setup(gateway_host, "dev", "magma_dev.yml")
         gateway_ip = gateway_host.split('@')[1].split(':')[0]
 
-    execute(_restart_gateway)
+    execute(_start_gateway)
```

Comment:
Replaceing a restart with a start seems wrong.

Reply:
Before this change the `_start_gateway` function included an unnecessary Make build. In all workflows this call is the first start of the AGW.

Reply:
Yepp but if you run it locally the agw could already be running

Reply:
This function is not meant for local use - if you want to use this locally it requires a number of manual steps first. Even if the AGW is already running the `_start_gateway` does not break anything and continues as intended. If you look at the `integ_test` function that uses Make you can see that it already used the `_start_gateway` function. The `_restart_gateway` was just a workaround for bazel to avoid the Make build.

Reply:
Ah ok this is inside the post build block